php-fpm/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:7.2-fpm
+
+RUN apt-get update
+
+RUN apt-get install -y zlib1g-dev libpq-dev git libicu-dev libxml2-dev \
+    && docker-php-ext-configure intl \
+    && docker-php-ext-install intl \
```

Comment:
The `intl` could be installed before `pdo` in the other `docker-php-ext-install` two lines below